.github/workflows/build-test.yml

```diff
@@ -270,6 +270,60 @@ jobs:
         crate: cargo-make
         version: latest
 
+    - name: Install Python
```

Comment:
Should this run only after verify-ffi?

Reply:
IMO no, i think this job is a lot like the fmt job: stuff is still likely to be working even if this fails, and it's easy to forget to run it and have it fail. It's good to have two statuses on each run, that's part of the goal behind splitting these.